crates/oxc_ast/src/ast/ts.rs

```diff
@@ -68,9 +68,19 @@ pub struct TSThisParameter<'a> {
 #[scope]
 #[derive(Debug)]
 #[generate_derive(CloneIn, Dummy, TakeIn, GetSpan, GetSpanMut, ContentEq, ESTree)]
+#[estree(
+    add_ts_def = "
+        interface TSEnumBody extends Span {
+            type: 'TSEnumBody';
+            members: TSEnumMember[];
+        }
+    ",
+    add_fields(body = TSEnumDeclarationBody)
```

Comment:
Thanks for getting involved!

I think you can achieve the same thing in a slightly cleaner way by:

* Removing `add_fields(body = TSEnumDeclarationBody)`.
* Removing `#[estree(skip)]` on `members` field.
* Instead, add this before `members` field:

```rs
#[scope(enter_before)]
#[estree(rename = "body", via = TSEnumDeclarationBody)]
pub members: Vec<'a, TSEnumMember<'a>>,
```

You shouldn't need to change the implementation of `TSEnumDeclarationBody` converter - it should work exactly as before.